# Topic Device Information

# ======== Variables Field ========

## Node name
string node_name

## Topic name.
string topic_name

## Topic qos type. Set to QOS_TYPE_XXX under TopicQosProfile.msg.
uint8 qos_type 0

## Manage QoS node.
string manage_qos_node
